src/extension.ts

```diff
@@ -6,16 +6,34 @@ import {
   ExtensionContext,
   DocumentFilter,
   TextDocument,
+  OutputChannel,
+  workspace,
   Position,
   env,
-  Uri
+  Uri,
+  ViewColumn
 } from 'vscode';
 
 import axios from 'axios';
+import { ImplementationGuideDependsOn } from 'fsh-sushi/dist/fhirtypes';
+import { readJSONorXML } from 'gofsh/dist/utils';
+import { fshToFhir } from 'fsh-sushi/dist/run';
+import { fhirToFsh } from 'gofsh/dist/api';
```

Comment:
This is indeed better. If I look in the package.json of the gofsh package it defines a main.
```
  "main": "dist/index.js",
```
This sets the primary entry point of the package. Could this be the reason for the ```from 'gofsh/dist'``` instead of ```from 'gofsh'```

Reply:
I _though_ that setting up the entry point like that is what would allow us to simply use `from 'gofsh'`. We do something similar in SUSHI [here](https://github.com/FHIR/sushi/blob/master/package.json#L40) and importing `from 'fsh-sushi'` seems to work right.

Ah-ha! I've just taken a better look. In SUSHI's package.json, we have this:
```
"types": "dist/index.d.ts",
```
But in GoFSH's we have this:
```
"types": "dist/app.d.ts",
```
I think that GoFSH pointing to `dist/app.d.ts` is a mistake. It should be `dist/index.d.ts`! We'll have to fix that for the next GoFSH release!